tests/probes/rpm/rpminfo/all.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
@evgenyz On my machine it starts to pass if I merge in #1431 . Do you experience the same situation?

Reply:
I have no problems with or without #1431. Strange

Reply:
That's strange. I should try it on a different machine. @matejak or @matusmarhefka or @ggbecker or @vojtapolasek do you have the same problem?

Reply:
@evgenyz I have tried it again with a clean build on my machine. I have also tried it in a fresh Fedora 31 VM. In both cases the output without #1431 applied is the same as pasted above. Could you check again?

Reply:
Here is mine. Clean build. F30. Can you please test the latest: I've removed `all.sh` and unified `/usr/bin/env` usage.

```
sudo  ctest --verbose -R probes/rpm/rpminfo
UpdateCTestConfiguration  from :/home/ekolesni/Devel/openscap/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/ekolesni/Devel/openscap/build/DartConfiguration.tcl
Test project /home/ekolesni/Devel/openscap/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 143
    Start 143: probes/rpm/rpminfo/test_probes_rpminfo.sh

143: Test command: /home/ekolesni/Devel/openscap/tests/probes/rpm/rpminfo/test_probes_rpminfo.sh
143: Environment variables: 
143:  srcdir=/home/ekolesni/Devel/openscap/tests/probes/rpm/rpminfo
143:  top_srcdir=/home/ekolesni/Devel/openscap
143:  builddir=/home/ekolesni/Devel/openscap/build
143: Test timeout computed to be: 10000000
143: + rpminfo probe test                                          
143: TEST: rpminfo probe test
143: Definition oval:1:def:9: false
143: Definition oval:1:def:8: true
143: Definition oval:1:def:7: false
143: Definition oval:1:def:6: true
143: Definition oval:1:def:5: true
143: Definition oval:1:def:4: false
143: Definition oval:1:def:3: true
143: Definition oval:1:def:2: false
143: Definition oval:1:def:13: false
143: Definition oval:1:def:12: false
143: Definition oval:1:def:11: true
143: Definition oval:1:def:10: false
143: Definition oval:1:def:1: false
143: Evaluation done.
143: RESULT: PASSED
143: 
1/2 Test #143: probes/rpm/rpminfo/test_probes_rpminfo.sh ...........   Passed    4.01 sec
test 144
    Start 144: probes/rpm/rpminfo/test_probes_rpminfo_offline.sh

144: Test command: /home/ekolesni/Devel/openscap/tests/probes/rpm/rpminfo/test_probes_rpminfo_offline.sh
144: Environment variables: 
144:  srcdir=/home/ekolesni/Devel/openscap/tests/probes/rpm/rpminfo
144:  top_srcdir=/home/ekolesni/Devel/openscap
144:  builddir=/home/ekolesni/Devel/openscap/build
144: Test timeout computed to be: 10000000
144: + rpminfo probe test (offline)                                
144: TEST: rpminfo probe test (offline)
144: Definition oval:1:def:9: false
144: Definition oval:1:def:8: true
144: Definition oval:1:def:7: false
144: Definition oval:1:def:6: true
144: Definition oval:1:def:5: true
144: Definition oval:1:def:4: false
144: Definition oval:1:def:3: true
144: Definition oval:1:def:2: false
144: Definition oval:1:def:13: false
144: Definition oval:1:def:12: false
144: Definition oval:1:def:11: true
144: Definition oval:1:def:10: false
144: Definition oval:1:def:1: false
144: Evaluation done.
144: RESULT: PASSED
144: 
2/2 Test #144: probes/rpm/rpminfo/test_probes_rpminfo_offline.sh ...   Passed    2.29 sec

The following tests passed:
	probes/rpm/rpminfo/test_probes_rpminfo.sh
	probes/rpm/rpminfo/test_probes_rpminfo_offline.sh

100% tests passed, 0 tests failed out of 2

Total Test time (real) =   6.31 sec
```

Reply:
Oh, hell! I'm testing without your *previous* patch! Of course!

Reply:
OK, I poke around it a little bit. I can reproduce your behaviour if I add `#export OSCAP_PROBE_RPMDB_PATH="$RPMTEST/var/lib/rpm/"` to the offline test's init function `rpm_prepare_offline`. If there is no such variable everything is OK. Can you please check if you have is set for some reason?

Reply:
It's not quite like that. I still can't reproduce `141: error: cannot open Packages database in ` and `141: E: oscap:     RPM: no dbpath has been set`.

Reply:
I'm looking into this right now. 

I have tried it again with the HEAD dcf7b1f027d16425472288e1c785260132455887 and it still happens. 

I use rpm-4.14.2.1-5.fc30.x86_64. 

There are 2 occurences of `OSCAP_PROBE_RPMDB_PATH`  in my repo, but they are only reading this variable:

```
src/OVAL/probes/probe/probe_main.c:294: if (getenv("OSCAP_PROBE_RPMDB_PATH") != NULL) {
src/OVAL/probes/unix/linux/rpminfo_probe.c:293: char *dbpath = getenv("OSCAP_PROBE_RPMDB_PATH");
```

Reply:
Anyway, we probably don't need to dive into this problem deeply, because we know that #1434 solves it for me, so we already have something that fixes. I was only curious why it works for you.

Reply:
Fixed. My machine setup used `macros` from different source.